src/OpenSage.Game/Logic/Object/Locomotor.cs

```diff
@@ -32,64 +32,80 @@ namespace OpenSage.Logic.Object
     ///   These control how much acceleration and cornering, respectively, will cause
     ///   the chassis to pitch or roll.
     /// </summary>
-    internal sealed class Locomotor
+    public sealed class Locomotor
     {
         private readonly GameObject _gameObject;
         private readonly LocomotorSet _locomotorSet;
         private readonly LocomotorTemplate _locomotorTemplate;
 
+        public float LiftFactor;
+
         public Locomotor(GameObject gameObject, LocomotorSet locomotorSet)
         {
             _gameObject = gameObject;
             _locomotorSet = locomotorSet;
             _locomotorTemplate = locomotorSet.Locomotor.Value;
-        }
-
-        public float GetPitchDamping()
-        {
-            return _locomotorTemplate.PitchDamping;
+            LiftFactor = 1.0f;
         }
 
         //TODO: check if the damaged values exists
-        private float GetAcceleration()
+        public float GetAcceleration()
         {
             return _gameObject.IsDamaged
-                ? GetLocomotorValue(x => x.AccelerationDamaged)
-                : GetLocomotorValue(x => x.Acceleration);
+                ? GetScaledLocomotorValue(_ => _.AccelerationDamaged)
```

Comment:
I prefer “_” is reserved for discarding variables. Here we actually use it, so let’s keep the name “x”.